android/app/build.gradle

```diff
@@ -41,6 +45,40 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
+    if (dropboxAppID) {
+        def dropboxActivity = """<activity
+                android:name="com.dropbox.core.android.AuthActivity"
+                android:configChanges="orientation|keyboard"
+                android:launchMode="singleTask">
+            <intent-filter>
+    
+            <!-- Change this to be db- followed by your app key -->
```

Comment:
@hristoterezov, the comment is unnecessary I think but, moreover, it's incorrect: the devs are not supposed to replace the app key here.

Also, is it a key or an ID according to Dropbox's docs? Our coding style requires one name per abstraction. And that's even more useful when facing third-party devs - we want to use the same working as Dropbox (or at least the prevailing one if Dropbox are using two names) so that we don't send third-party looking for both key and ID in Dropbox's docs and scratching their heads please!

Reply:
Agreed! I don't think they are too consistent in the naming about this - they have app id, app key, client id, app id, etc. I don't think I was very consistent in the PR as well. But I'll check it out and come up with a consistent name.